packages/react-ui-ag/src/Gmap/Gmap.js

```diff
@@ -0,0 +1,24 @@
+import React, { PureComponent } from 'react'
+import PropTypes from 'prop-types'
+import { Gmap as GoogleMap, GmapSpinner } from '@rentpath/react-ui-core'
+
+export default class Gmap extends PureComponent {
+  static propTypes = {
+    apiKey: PropTypes.string.isRequired,
+  }
+
+  render() {
+    const {
+      apiKey,
+      ...rest
+    } = this.props
+
+    return (
+      <GoogleMap
+        apiKey={apiKey}
+        spinner={<GmapSpinner color="#ff0000" loading closeDelay={1000} />}
```

Comment:
don't need to make `closeDelay` configurable?

Reply:
I think the idea is that this is the configurable part - if we wanted to update the GmapSpinner for ag or rent, we would just update it here in there respective Gmap files - this would also apply to adding styles/classes etc, I believe